specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/examples/CreateOrUpdateADedicatedHost.json.orig

```diff
@@ -0,0 +1,68 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2018-10-01",
+    "parameters": {
+      "location": "westus",
+      "tags": {
+        "department": "HR"
+      },
+      "properties": {
+        "platformFaultDomain": 1
+      },
+      "sku": {
+        "name": "DSv3-Type1"
+      }
+    },
+    "hostGroupName": "myDedicatedHostGroup",
+    "hostName": "myDedicatedHost"
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "name": "myDedicatedHost",
+        "location": "westus",
+        "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/HostGroups/myDedicatedHostGroup/hosts/myDedicatedHost",
+        "tags": {
+          "department": "HR"
+        },
+        "sku": {
+          "name": "DSv3-Type1"
+        },
+        "properties": {
+          "platformFaultDomain": 1,
+          "autoReplaceOnFailure": true,
+          "hostId": "{GUID}"
+        }
+      }
+    },
+<<<<<<< HEAD
```

Comment:
can you do merge integration?

Reply:
I used "git mergetool" and do the merging however it does seem to have worked. How can I fix it?

Reply:
Reverted